metaflow/plugins/kfp/kfp_utils.py

```diff
@@ -332,3 +332,11 @@ def _upload_pipeline(
         print(f"Uploaded test pipeline {pipeline_id} version {version_id}.")
 
     return pipeline_id, version_id
+
+
+def get_notebook_metaflow_sa() -> str:
+    """Returns the notebook service account from individual profiles if a user
+    inputs an IAM role for their notebook. If no ServiceAccount is set in the notebook
+    return the `deafult-editor` ServiceAccount which is default in all profiles.
+    """
+    return os.environ.get("METAFLOW_KUBERNETES_SERVICE_ACCOUNT", "default-editor")
```

Comment:
I understand that this is still in DRAFT mode 👍 

This could live in https://github.com/zillow/metaflow/blob/f150287e9d37bf66e7977a8945be1e16d687005c/metaflow/metaflow_config.py#L158 and be named something like `KFP_SERVICE_ACCOUNT_NAME`.

Reply:
I'm assuming that k8s only allows a user to select a service account name in their current **namespace**? (to disallow someone assuming a service account they do not have access to)

Reply:
ServiceAccounts are namspace level resources so they cannot be access from other namespaces. Also I moved this based on a convo with Yun.